src/components/sections/items/ToggleSectionItem.tsx

```diff
@@ -9,10 +9,12 @@ import {useSectionStyle} from '../use-section-style';
 import {Toggle} from '@atb/components/toggle';
 import {InteractiveColor} from '@atb/theme/colors';
 import {SvgProps} from 'react-native-svg';
+import {InfoChip} from '@atb/components/info-chip';
 
 type Props = SectionItemProps<{
   text: string;
   subtext?: string;
+  infoChipLabel?: string;
```

Comment:
In the `LinkSectionItem` we have essentially the same functionality, with a field named `flag`. I think the two implementations should be consolidated, so both the field naming and the code in the section items are pretty much the same, and they both support both "new" and "beta".